build.gradle

```diff
@@ -202,8 +202,3 @@ buildscript {
         classpath 'org.eclipse.jgit:org.eclipse.jgit:2.0.0.201206130900-r'
     }
 }
-
-task wrapper(type: Wrapper) {
```

Comment:
@JesusFreke Okay, I added a comment with the command line syntax for the Gradle Wrapper upgrade in `build.gradle`. Would you be OK with that?

I don't think it's a good idea to override the included `wrapper` task with a custom one with the same name:
- It removes the documentation and any (potentially) added functionality of the included one.
- It requires to open, edit, *and (unnecessarily) commit* `build.gradle` for each wrapper upgrade.
- The Gradle wrapper version has to be kept in sync in both `build.gradle` **and** `gradle/wrapper/gradle-wrapper.properties`